packages/cli/src/cog/cutline.ts

```diff
@@ -21,11 +21,32 @@ function polysSameShape(a: Polygon, b: Polygon): boolean {
     return true;
 }
 
+function namedBounds(tms: TileMatrixSet, tile: Tile): NamedBounds {
+    return { name: TileMatrixSet.tileToName(tile), ...tms.tileToSourceBounds(tile).toJson() };
+}
+
+function contains(a: BoundingBox, b: BoundingBox): boolean {
+    return a.x <= b.x && a.x + a.width >= b.x + b.width && a.y <= b.y && a.y + a.height >= b.y + b.height;
+}
+
+function addNonDupes(list: Tile[], addList: Tile[]): void {
+    const len = list.length;
+    loop: for (const add of addList) {
+        for (let i = 0; i < len; ++i) {
+            const curr = list[i];
+            if (curr.x == add.x && curr.y == add.y && curr.z == add.z) {
+                continue loop;
```

Comment:
dont use goto's

Reply:
And you reasoning is?

Reply:
we have 0 gotos in the codebase right now, this would be changing the convention of not using them.